src/meta/proto-conv/tests/it/user_stage.rs

```diff
@@ -43,6 +43,55 @@ fn test_user_stage_gcs_latest() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[test]
+fn test_user_stage_s3_v8() -> anyhow::Result<()> {
+    // Encoded data of version 8 of user_stage_s3:
+    // It is generated with common::test_pb_from_to.
+    let user_stage_s3_v8 = vec![
```

Comment:
The new `VER` value is `9`, I think the test to add should be suffixed with `_v9`?

Reply:
Tracked in https://github.com/datafuselabs/databend/issues/7770